docs/source/lr_finder.rst

```diff
@@ -0,0 +1,53 @@
+Learning Rate Finder
+--------------------
+
+For training deep neural networks, selecting a good learning rate is essential
+for both better performance and faster convergence. Even optimizers such as
+`Adam` that are self-adjusting the learning rate can benefit from more optimal
+choices.
+
+To reduce the amount of guesswork conserning choosing a good initial learning
+rate, a learning rate finder can be used. As described in this (paper)[https://arxiv.org/abs/1506.01186], 
+a lr finder does a small run where the lr is increased after each processed
+batch and the corresponding loss is logged. The result of this is therefore a
+`lr` vs. `loss` plot that can be used as guidence for choosing a optimal
+initial lr. 
+
+Using Lightnings build in LR finder
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A typical use of the build in lr finder would look something like this. It is not
+recommended to pick the lr with the lowest loss, but instead choosing a point 
+point corresponds to 
```

Comment:
did you forget to finish the sentence here? assuming you meant to say something like "corresponds to the steepest decrease in loss"